Atmel/LED Blinking/led_blinking.c

```diff
@@ -0,0 +1,15 @@
+#include <avr/io.h>
+#define F_CPU 16000000L
+#include <util/delay.h>
+
+int main(void)
+{
+	DDRB = 0xff;
+	while (1)
+	{
+		PORTB = 0xff;
+		_delay_ms(1000);
+		PORTB = 0x00;
+		_delay_ms(1000);
+	}
+}
```

Comment:
Add New Line at End of File (EOF)

Reply:
@Harshil-Jani  in separate commit ?

Reply:
Anything that suits you. You can rebase and do the changes in same commit or maybe create a new commit and then squash all of them together.

Reply:
@Harshil-Jani Done.